corehq/apps/cloudcare/dbaccessors.py

```diff
@@ -20,7 +20,7 @@ def get_application_access_for_domain(domain):
 
 def get_cloudcare_apps(domain):
     return [
-        app for app in ApplicationBase.get_db().view(
+        app for app in ApplicationBase.view(
             'app_manager/applications_brief', startkey=[domain], endkey=[domain, {}])
         if app['doc_type'] == 'Application' and app['cloudcare_enabled']
```

Comment:
Looks like the wrapped docs actually support `__get_item__()`, but would be cleaner to use dot notation.